crates/stateless/src/fork_spec.rs

```diff
@@ -1,8 +1,11 @@
+use alloy_eips::{eip6110::MAINNET_DEPOSIT_CONTRACT_ADDRESS, BlobScheduleBlobParams};
 // This is here so we don't pull in the EF-tests.
```

Comment:
In the future, we can probably remove this entirely and have a From impl in ef-tests crate that converts from the old ForkSpec defined there into the new ChainSpec